app/vmagent/influx/request_handler.go

```diff
@@ -50,11 +50,12 @@ func InsertHandlerForHTTP(at *auth.Token, req *http.Request) error {
 		return err
 	}
 	isGzipped := req.Header.Get("Content-Encoding") == "gzip"
+	isStreamMode := req.Header.Get("Stream-Mode") == "1"
```

Comment:
AFAIK, you can set GET params easily in http output:
```
[[outputs.http]]
  url = "http://localhost:8428/api/v1/write?foo=1"
```
it worked for me, at least.

Reply:
but for `outputs.influxdb`, which is mentioned in our docs it's not available

Reply:
It also, would be great to create a switch between default behavior. 

E.g. with flag `-influx.disableStreamParse`, parser will use batch parsing by default.

 It allows to control parsing without changes to the clients.

Reply:
Ah, sorry. New implementation is actually changes default behavior. In this case, flag is not needed.